docs/index.md

```diff
@@ -0,0 +1,397 @@
+---
```

Comment:
I copy-pasted our `README` here. We can iterate after we've verified that everything works.